src/settings/SettingTab.tsx

```diff
@@ -495,6 +495,42 @@ export class SmartCopilotSettingTab extends PluginSettingTab {
           }),
       )
 
+    new Setting(containerEl)
+      .setName('Include patterns')
+      .setDesc(
+        'If any patterns are specified, ONLY files matching at least one pattern will be included in indexing. One pattern per line. Uses glob patterns (e.g., "notes/*", "*.md"). Leave empty to include all files not excluded by exclude patterns. After changing this, use the command "Rebuild entire vault index" to apply changes.',
+      )
```

Comment:
I think it would be better to have include option first before exclude option because it feels more natural to think in order of "what to include" then "what to remove from included items